readme.md

```diff
@@ -18,16 +18,69 @@ pod 'MapboxSpeech', '~> 0.1'
 
 Then `import MapboxSpeech` or `@import MapboxSpeech;`.
 
-## Basic Usage
+## Usage
+
+You’ll need a [Mapbox access token](https://www.mapbox.com/developers/api/#access-tokens) in order to use the API. If you’re already using the [Mapbox Maps SDK for iOS](https://www.mapbox.com/ios-sdk/) or [macOS SDK](https://mapbox.github.io/mapbox-gl-native/macos/), Mapbox Speech automatically recognizes your access token, as long as you’ve placed it in the `MGLMapboxAccessToken` key of your application’s Info.plist file.
+
+The examples below are each provided in Swift (denoted with `main.swift`) and Objective-C (`main.m`).
+
+### Basics
+
+The main speech synthesis class is SpeechSynthesizer (in Swift) or MBSpeechSynthesizer (in Objective-C). Create a speech synthesizer object using your access token:
 
 ```swift
 // main.swift
 import MapboxSpeech
 
-let voice = SpeechSynthesizer(accessToken: "Your Mapbox access token")
-let options = SpeechOptions(text: "hello, my name is Bobby")
+let speechSynthesizer = SpeechSynthesizer(accessToken: "<#your access token#>")
+```
+
+```objc
+// main.m
+@import MapboxSpeech;
 
-voice.audioData(with: options) { (data: Data?, error: NSError?) in
+MBSpeechSynthesizer *speechSynthesizer = [[MBSpeechSynthesizer alloc] initWithAccessToken:@"<#your access token#>"];
+```
+
+Alternatively, you can place your access token in the `MGLMapboxAccessToken` key of your application’s Info.plist file, then use the shared speech synthesizer object:
```

Comment:
Actually, this documentation doesn’t really prefer `shared` over the initializer. It’s an alternative. Does that allay your concerns about proliferating this design pattern for the time being, @akitchen?